docs/extensions.adoc

```diff
@@ -439,6 +439,19 @@ annotated object. To prevent cleanup exceptions from being reported, override th
 def ignoreMyExceptions
 ----
 
+=== TempDir
+
+To generate a temp directory for test and delete it after test,
+annotate a member field of type `java.io.File`, `java.nio.file.Path`
+or untyped using `def` in a spec class. If the annotated field is shared,
```

Comment:
You should mention that, `def` will inject a `Path`.

Reply:
q.e.d. it is mentioned in the code comment of the example.
That's why I also asked for the other descirptions to be done in the text, not in code comments. :-)